src/contentmanagermodel.cpp

```diff
@@ -101,3 +99,43 @@ QVariant ContentManagerModel::headerData(int section, Qt::Orientation orientatio
         default: return QVariant();
     }
 }
+
+void ContentManagerModel::setBooksData(const QList<QMap<QString, QVariant>>& data)
+{
+    m_data = data;
```

Comment:
Indeed.
In fact, we should not store a `m_data` but directly reference a `kiwix::Book`. But it will be for the refactoring.
Let's keep `m_data` for now.